src/shared/tpm2-util.h

```diff
@@ -67,6 +67,8 @@ typedef struct {
         void *tcti_dl;
         TSS2_TCTI_CONTEXT *tcti_context;
         ESYS_CONTEXT *esys_context;
+
```

Comment:
maybe add a comment here: `/* Some selected cached capabilities of the TPM */

Reply:
ack